ertviz/views/paremeter_view.py

```diff
@@ -47,6 +47,14 @@ def parameter_view(parent):
                             ],
                             value=["hist", "kde"],
                         ),
+                        html.Label("#Bins:"),
```

Comment:
If available space - would it be nicer with `number of bins` ?

Reply:
Wanted to save space, will check if the plot shrinks

Reply:
The plot didn't shrink, so changed.